ui/lib/kmip/addon/templates/credentials/index.hbs

```diff
@@ -64,21 +64,26 @@
       <Item.content>
         <Icon @name="file-text" class="has-text-grey-light" />{{list.item.id}}
       </Item.content>
-      <Item.menu as |m|>
+      <Item.menu>
         <li class="action">
           <LinkTo @route="credentials.show" @models={{array this.scope this.role list.item.id}} class="is-block">
             View credentials
           </LinkTo>
         </li>
         {{#if list.item.deletePath.canDelete}}
-          <MenuLoader @loadingParam={{list.item.deletePath.isPending}}>
-            <m.Message
-              @id={{list.item.id}}
-              @triggerText="Revoke credentials"
-              @title="Revoke this?"
-              @message="Any client using these credentials will no longer be able to."
-              @confirmButtonText="Revoke"
-              @onConfirm={{action
+          {{#if list.item.deletePath.isPending}}
```

Comment:
The loading dropdown will never load, right? Because `canDelete` will not be true until `isPending` finishes

Reply:
Yeah - that's a great point. I copied the original logic from `MenuLoader` - I'll amend this and the other spots where this happens